pkg/core/region_test.go

```diff
@@ -839,15 +839,15 @@ func newRegionInfoIDRandom(idAllocator id.Allocator) *RegionInfo {
 	// Randomly select a peer as the leader.
 	leaderIdx := mrand.Intn(peerNum)
 	for i := range peerNum {
-		id, _ := idAllocator.Alloc()
+		id, _, _ := idAllocator.Alloc(1)
```

Comment:
A lot of places are using `idAllocator.Alloc(1)`, what about providing a function named `AllocOne()`?

Reply:
Then we need to add one more interface.